src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java

```diff
@@ -48,13 +48,7 @@ public UUID send(String caseId, String userToken, String letterType, List<Docume
 
         String s2sToken = authTokenGenerator.generate();
         final List<String> stringifiedDocuments = documents.stream()
-            .map(docInfo -> {
-                try {
-                    return getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken);
-                } catch (IOException e) {
-                    throw new RuntimeException(e);
-                }
-            })
+            .map(docInfo -> getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken))
```

Comment:
same question as this - https://github.com/hmcts/prl-cos-api/pull/1813#discussion_r1373908029